web/src/components/FilterToggle.tsx

```diff
@@ -30,10 +38,12 @@ const FilterToggle = ({
 }): ReactElement => {
   const { t } = useTranslation('events')
   return (
-    <StyledButton label='toggleDate' onClick={() => setToggleDateFilter(!isDateFilterActive)}>
-      <Icon src={isDateFilterActive ? CloseFullscreenIcon : FilterListIcon} />
-      <Text>{t(isDateFilterActive ? 'hideFilters' : 'showFilters')}</Text>
-    </StyledButton>
+    <StyledAccordionSummary onClick={() => setToggleDateFilter(!isDateFilterActive)}>
+      <StyledContainer>
+        <StyledIcon src={isDateFilterActive ? CloseFullscreenIcon : FilterListIcon} />
+        <StyledTypography variant='button'>{t(isDateFilterActive ? 'hideFilters' : 'showFilters')}</StyledTypography>
+      </StyledContainer>
+    </StyledAccordionSummary>
```

Comment:
❓ Why don't we just use a mui `Button` here?
```suggestion
    <StyledAccordionSummary onClick={() => setToggleDateFilter(!isDateFilterActive)}>
      <Button startIcon={isDateFilterActive ? <CloseFullscreenIcon /> : <FilterListIcon />}>
        {t(isDateFilterActive ? 'hideFilters' : 'showFilters')}
      </Button>
    </StyledAccordionSummary>
```

Reply:
AccordionSummary is a button and putting a button inside another one will upset the DOM.
<img width="936" height="417" alt="image" src="https://github.com/user-attachments/assets/c72359d9-f08c-4887-bc05-5c632c8d73b7" />

Reply:
Good catch. Rendering the button as `div` works though and applies all the correct styles/effects:
``` tsx
    <StyledAccordionSummary onClick={() => setToggleDateFilter(!isDateFilterActive)}>
      <Button component='div' startIcon={isDateFilterActive ? <CloseFullscreenIcon /> : <FilterListIcon />}>
        {t(isDateFilterActive ? 'hideFilters' : 'showFilters')}
      </Button>
    </StyledAccordionSummary>
```